src/modules/components/asset/assetTransfer/assetTransfer.tsx

```diff
@@ -0,0 +1,150 @@
+import classNames from 'classnames';
+import { type Hash } from 'viem';
+import { useConfig } from 'wagmi';
+import { Avatar, AvatarIcon, IconType, NumberFormat, formatterUtils } from '../../../../core';
+import { type IWeb3ComponentProps } from '../../../types';
+import { AssetTransferAddress } from './assetTransferAddress';
+
+export interface IAssetTransferProps extends IWeb3ComponentProps {
+    /**
+     * Address of the transaction sender.
+     */
+    senderAddress: Hash;
+    /**
+     * Address of the transaction recipient.
+     */
+    recipientAddress: Hash;
+    /**
+     * ENS name of the transaction sender.
+     */
+    senderEnsName?: string;
+    /**
+     * ENS name of the transaction recipient.
+     */
+    recipientEnsName?: string;
+    /**
+     * Name of the token transferred.
+     */
+    tokenName: string;
+    /**
+     * Icon URL of the tranferred token.
+     */
+    tokenIconSrc?: string;
+    /**
+     * Amount of tokens transferred.
+     */
+    amount: number | string;
+    /**
+     * Symbol of the token transferred. Example: ETH, DAI, etc.
+     */
+    symbol: string;
+    /**
+     * Price per token in fiat.
+     */
+    fiatPrice?: number | string;
+    /**
+     * Transaction hash.
+     */
+    hash: string;
+    /**
+     * Chain ID of the transaction.
+     */
+    chainId: number;
+}
+
+const formatValue = (amount?: number | string, tokenSymbol: string = '', fiatPrice?: number | string) => {
+    const formattedTokenValue = formatterUtils.formatNumber(amount, {
+        format: NumberFormat.TOKEN_AMOUNT_SHORT,
+        withSign: true,
+        fallback: '-',
+    });
+    const fiatValue = Number(amount) * Number(fiatPrice);
+    const formattedFiatValue = formatterUtils.formatNumber(fiatValue, {
+        format: NumberFormat.FIAT_TOTAL_SHORT,
+        fallback: ` `,
+    });
+    const formattedTokenAmount = `${formattedTokenValue} ${tokenSymbol}`;
+
+    return { formattedTokenAmount, formattedFiatValue };
+};
+
+export const AssetTransfer: React.FC<IAssetTransferProps> = (props) => {
+    const {
+        senderAddress,
+        recipientAddress,
+        senderEnsName,
+        recipientEnsName,
+        tokenName,
+        tokenIconSrc,
+        amount,
+        symbol,
+        fiatPrice,
+        chainId,
+        hash,
+        wagmiConfig: wagmiConfigProps,
+    } = props;
+
+    const wagmiConfigProvider = useConfig();
+
+    const wagmiConfig = wagmiConfigProps ?? wagmiConfigProvider;
+
+    const processedChainId = chainId ?? wagmiConfig.chains[0].id;
+
+    const currentChain = wagmiConfig.chains.find(({ id }) => id === processedChainId);
+    const blockExplorerUrl = currentChain?.blockExplorers?.default.url;
```

Comment:
I did with this with useChains() in the TransactionDataListItem.Structure. I think this has a better fallback and scoped optionality for external users. But useChains() is also essentially bound to the config context for supported chain info from what I understand.